securedrop/tests/test_unit_journalist.py

```diff
@@ -7,15 +7,17 @@
 import mock
 
 from flask_testing import TestCase
-from flask import url_for, escape, g
+from flask import url_for, escape
 
 import crypto_util
 import journalist
 import common
-from db import db_session, Source, Journalist, InvalidPasswordLength
+from db import (db_session, Source, Journalist, InvalidPasswordLength,
+                Submission)
 
 # Set environment variable so config.py uses a test environment
 os.environ['SECUREDROP_ENV'] = 'test'
+import config
```

Comment:
>Currently, the securedrop/config.py modules reads the environment variable SECUREDROP_ENV, and sets the configuration differently based on its value.

Ok awesome. Could I suggest leaving a comment explaining that its needed even though it is not directly called?

>We'd like to deprecate the reading/ setting of environment variables in our code entirely, and instead be read from static INI files.

👍 makes complete sense. Only thing to consider is that environment variables are much easier to work with when it comes to Docker / containers than INI files in my experience.